custom_connectors/custom_auth/eventbrite_connector.rb

```diff
@@ -0,0 +1,104 @@
+{
+  title: "Eventbrite (custom)",
+
+  connection: {
+    fields: [
+      {
+        name: "personal_token",
+        control_type: "password",
+        hint: "Find your personal token here: https://www.eventbrite.com/myaccount/apps/",
+        optional: false
+      }
+    ],
+
+    authorization: {
+      type: "custom_auth",
+
+      apply: lambda do |connection|
+        headers("Authorization": "Bearer #{connection["personal_token"]}")
+      end
+    },
+
+    base_uri: lambda do
```

Comment:
Does this have to be lambda can this be a string? If it can be use string.

Reply:
It has to be a lambda